spec/system/admin/bulk_url_upload_spec.rb

```diff
@@ -16,9 +16,10 @@
   include ActiveJob::TestHelper
 
   subject(:do_bulk_upload) do
-    perform_enqueued_jobs do
+    perform_enqueued_jobs except: SearchgovDomainIndexerJob do
```

Comment:
This ensures we run the jobs associated with the bulk upload functionality, but not actually start indexing the URLs, as that causes unrelated spec failures.